src/components/AttachmentPicker/index.js

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
-import propTypes from './AttachmentPickerPropTypes';
+import CONST from '../../CONST';
+import * as attachmentPickerPropTypes from './AttachmentPickerPropTypes';
+
+/**
+ * Returns acceptable FileTypes based on ATTACHMENT_PICKER_TYPE
+ * @param {String} type
+ * @returns {String|undefined} file types string
+ */
+function getAcceptableFileTypes(type) {
+    if (type === CONST.ATTACHMENT_PICKER_TYPE.IMAGE) {
+        return 'image/*';
+    }
+
+    // Accept all file types when its undefined
```

Comment:
NAB (not a blocker), but this comment could probably be moved up to the JSDoc for `@returns`

Reply:
Updated